packages/api/src/store/webhook-table.ts

```diff
@@ -49,12 +49,14 @@ export default class WebhookTable extends Table<DBWebhook> {
   }
 
   async updateStatus(id: string, status: DBWebhook["status"]) {
+    const statusStr = JSON.stringify(status);
     const res = await this.db.query(
-      `UPDATE ${
-        this.name
-      } SET data = jsonb_set(data, '{status}', case when data->'status' is null then '{}' else data->'status' end || '${JSON.stringify(
-        status
-      )}') WHERE id = '${id}'`
+      sql``
```

Comment:
Is this change related to the local IP change? Or is it just refactor? Both are fine, I'm just trying to connect the dots.

Reply:
It is not related, it is a fix to use proper SQL parameters in the query instead of raw string interpolation. As I was testing this there were some errors that contained `'` and the query failed, so we need to use proper `sql\`\`` strings here. I can move this to a separate PR if you think it's necessary.